Documentation/Getting-Started-For-Windows.md

```diff
@@ -15,6 +15,7 @@ may be replaced in all commands below with the directory of your choice.
 * Clone and build CoreCLR:
     ```
     $ git clone https://github.com/dotnet/coreclr
+    $ git clone https://github.com/dotnet/corefx
```

Comment:
No, not as far as I know. I remember there being some issue with nuget last week (@mmitche would have more info), but in the lab we do not build corefx locally, so it should not be necessary.